### PR TITLE
register LSU - Deep Bayou Hosted CE

### PR DESCRIPTION
**Pull request by jdost321**
Creating with active set to false, until we're ready to push to production

### CODE REVIEW

File: topology/Louisiana State University/LSU - Deep Bayou/LSU-Deep_Bayou.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Production: true
+SupportCenter: Self Supported
+GroupID: 1105
+GroupDescription: This is the LSU - Deep Bayou cluster
+Resources:
+  LSU-Deep_Bayou:
```
Comment (brianhlin) [inline topology/Louisiana State University/LSU - Deep Bayou/LSU-Deep_Bayou.yaml:6]:
```suggestion
  LSU-DB-CE1:
```

Maybe something like this to more closely match the first bit of the FQDN?

**Review by brianhlin — APPROVED** [review]
One comment on the resource name but otherwise LGTM

File: topology/Louisiana State University/LSU - Deep Bayou/LSU-Deep_Bayou.yaml (hunk 1 above)
Comment (jdost321) [reply topology/Louisiana State University/LSU - Deep Bayou/LSU-Deep_Bayou.yaml:6]: Hmm I think I like this suggestion. @jeffreynpeterson what are your thoughts? We can update the naming convention document if you agree.

Comment (jeffreynpeterson) [reply topology/Louisiana State University/LSU - Deep Bayou/LSU-Deep_Bayou.yaml:6]: matching the FQDN is good, we will still just have to make the call as to when to shorten or not the name for that depending on site.

**Pull request merged**